mmv1/third_party/terraform/services/compute/data_source_google_compute_ha_vpn_gateway_test.go

```diff
@@ -42,3 +55,23 @@ data "google_compute_ha_vpn_gateway" "ha_gateway" {
 }
 `, gwName, gwName)
 }
+
+func testAccDataSourceComputeHaVpnGatewayFields(gwName, gatewayIpVersion, stackType string) string {
+	return fmt.Sprintf(`
+resource "google_compute_ha_vpn_gateway" "ha_gateway" {
+  name     			 = "%s"
+  network  			 = google_compute_network.network1.id
+  gateway_ip_version = "%s"
+  stack_type		 = "%s"
+}
+
+resource "google_compute_network" "network1" {
+  name                    = "%s"
+  auto_create_subnetworks = false
+}
+
+data "google_compute_ha_vpn_gateway" "ha_gateway" {
+  name = google_compute_ha_vpn_gateway.ha_gateway.name
+}
+`, gwName, gateway_ip_version, stack_type, gwName)
```

Comment:
these values don't match the variable parameters of the function, leading to the error below:

```
# github.com/hashicorp/terraform-provider-google-beta/google-beta/services/compute_test [github.com/hashicorp/terraform-provider-google-beta/google-beta/services/compute.test]
google-beta/services/compute/data_source_google_compute_ha_vpn_gateway_test.go:78:12: undefined: gateway_ip_version
google-beta/services/compute/data_source_google_compute_ha_vpn_gateway_test.go:78:32: undefined: stack_type
```